File: Python_programming/Week_9_from_41_to_46/A3.py
#!/usr/bin/python3

age = int(input("What is your age?: ").strip())

if age > 10 and age < 100:
    age_in_years = age
    age_in_month = age_in_years * 12
    age_in_weeks = age_in_month * 4
    age_in_days = age_in_years * 365
    age_in_min = age_in_weeks * 7 * 24
    age_in_sec = age_in_min * 60


    print(f"You are {age_in_month:,} in months")
    print(f"You are {age_in_month:,} in weeks")
    print(f"You are {age_in_days:,} in days")
    print(f"You are {age_in_min:,} in min")
    print(f"You are in {age_in_sec:,} seconds")
else:
    print("Age is out of range")
    exit

File: Front_end_dev/JAVASCRIPT/JAVASCRIPT_Assignments/Lessons[57-63]/A3.js
function ageInTime(theAge) {
  if (theAge > 10 && theAge < 100) {
    console.log(`You have ${theAge} in years`)
    console.log(`You have ${theAge * 12} in months`)
    console.log(`You have ${(theAge * 12) * 30} in days`)
    console.log(`You have ${((theAge * 12) * 30) * 24} in hours`)
    console.log(`You have ${(((theAge * 12) * 30) * 24) * 60} in minutes`)
    console.log(`You have ${((((theAge * 12) * 30) * 24) * 60) * 60} in seconds`)
  } else {
    console.log(`Your age is out of range`)
  }
}

// Needed Output
ageInTime(110); // Age Out Of Range
ageInTime(38); // Months Example => 456 Months